.env.example

```diff
@@ -23,5 +23,11 @@ export CELESTIA_NODE_URL="http://localhost:26658"
 export CELESTIA_NODE_AUTH_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.qxoN_a-k7aZZE77VSpfgyvI_0eNvJ_OgBsT35zxUQnQ"
 # Namespace of the rollup on Celestia Mainnet
 export CELESTIA_NAMESPACE="0f0f0f0f0f0f0f0f0f0f"
-# RPC url of the simapp simulating Celestia 
+# RPC url of the simapp simulating Celestia
 export SIMAPP_RPC_URL=grpc://127.0.0.1:9190
+# RPC url of the EVM rollup
+export EVM_RPC_URL=http://localhost:8545
```

Comment:
As a FLUP we can de-duplicate this with RPC_URL which is above

Reply:
Followed up in #252